Sources/Components/ARCTitleImageCard.swift

```diff
@@ -30,31 +37,69 @@ public struct ARCTitleImageCard<Trailing: View>: View {
 
     public var body: some View {
         VStack(spacing: 0) {
-            Text(verbatim: title)
-                .style(.arcH2)
-                .foregroundColor(.arcBlack)
-            Text(verbatim: subtitle)
-                .style(.arcH4)
-                .foregroundColor(.arcDarkGray)
-                .padding(.top, .ArcTitleImageCard.subtitlePaddingTop)
-            image
-                .resizable()
-                .scaledToFill()
-                .frame(height: .ArcTitleImageCard.imageHeight)
-                .frame(maxWidth: .infinity)
-                .cornerRadius(.arcCornerRadius)
-                .padding(.top, .arcVerticalPadding)
+            TitleImageCardHeader(
+                title: title,
+                subtitle: subtitle,
+                isPortrait: isPortrait
+            )
+            if isPortrait {
+                image
+                    .resizable()
+                    .scaledToFill()
+                    .frame(maxWidth: .infinity, maxHeight: .ArcTitleImageCard.imageHeight)
+                    .cornerRadius(.arcCornerRadius)
+                    .padding(.top, .arcVerticalPadding)
+            }
             trailing()
                 .padding(.top, .arcVerticalPadding)
+                .frame(maxWidth: .infinity)
+                .arcSeparator()
         }
-        .padding(.ArcTitleImageCard.padding)
-        .padding(.top, .arcVerticalPadding)
+        .padding(.horizontal, isPortrait ? .arcVerticalPadding : 0)
+        .padding(.vertical, .arcVerticalPadding)
         .multilineTextAlignment(.center)
         .background(Color.arcWhite)
         .cornerRadius(.arcCornerRadius)
     }
 }
 
+// MARK: - TitleImageCardHeader
+
+private struct TitleImageCardHeader: View {
+
+    var title: String
+    var subtitle: String
+    var isPortrait: Bool
+
+    var body: some View {
+        if isPortrait {
+            VStack(spacing: 0) {
+                Text(verbatim: title)
+                    .style(.arcH2)
+                    .foregroundColor(.arcBlack)
+                Text(verbatim: subtitle)
+                    .style(.arcH4)
+                    .foregroundColor(.arcDarkGray)
+                    .padding(.top, .ArcTitleImageCard.subtitlePaddingTop)
+            }
```

Comment:
Do we want any multline text alignment stuff btw? Might not but just checking

Reply:
Center is default of multiline alignement and in portrait we want it centered 😊